examples/traffic-splitting/README.md

```diff
@@ -0,0 +1,77 @@
+# Example
+
+In this example we will deploy NGINX Kubernetes Gateway and configure traffic splitting for a simple cafe application.
+We will use `HTTPRoute` resources to split traffic between two versions of the application -- `coffee-v1` and `coffee-v2`.
+
+## Running the Example
+
+## 1. Deploy NGINX Kubernetes Gateway
+
+1. Follow the [installation instructions](/docs/installation.md) to deploy NGINX Gateway.
+
+1. Save the public IP address of NGINX Kubernetes Gateway into a shell variable:
+
+   ```
+   GW_IP=XXX.YYY.ZZZ.III
+   ```
+
+1. Save the port of NGINX Kubernetes Gateway:
+
+   ```
+   GW_PORT=<port number>
+   ```
+
+## 2. Deploy the Coffee Application
+
+1. Create the Cafe Deployments and Services:
+
+   ```
+   kubectl apply -f cafe.yaml
+   ```
+
+1. Check that the Pods are running in the `default` namespace:
+
+   ```
+   kubectl -n default get pods
+   NAME                         READY   STATUS    RESTARTS   AGE
+   coffee-v1-7c57c576b-rfjsh    1/1     Running   0          21m
+   coffee-v2-698f66dc46-vcb6r   1/1     Running   0          21m
+   ```
+
+## 3. Configure Routing
+
+1. Create the `Gateway`:
+
+   ```
+   kubectl apply -f gateway.yaml
+   ```
+
+1. Create the `HTTPRoute` resources:
+
+   ```
+   kubectl apply -f cafe-route.yaml
+   ```
+   
+This `HTTPRoute` resource defines a route for the path `/coffee` that sends 80% of the requests to `coffee-v1` and 20% to `coffee-v2`.
+
+## 4. Test the Application
```

Comment:
What do either of you think about a #5 step to show off dynamic updates?

In 5 we have them change the relative proportions and test again to see the traffic responses shift. This is already covered in the public guide, but they may find us from a different path.

Reply:
Sounds good to me. I'll add it

Reply:
Done